pkg/packageGenerator/javascript/generator.go

```diff
@@ -65,7 +65,7 @@ func (g *Generator) GetPackageName() string {
 
 func (g *Generator) PushPackage(packageDir string) error {
 	out, err := g.Cmd.Execute(packageDir, "npm", "publish")
-	log.Logger().Infof(out)
+	log.Logger().Infof("%s", out)
```

Comment:
The linter was failing due to string formatting being called but not needed. You can just remove the f.
```suggestion
	log.Logger().Info(out)
```